dashboard/config/database.yml

```diff
@@ -65,6 +66,16 @@ environment_defaults: &environment_defaults
     port: <%= reader.port || 3306 %>
     username: <%= reader.user %>
   <% end %>
+  <% if reporting_reader %>
```

Comment:
Should I have this `if reporting_reader`? If I should, should `get_reporting_role_name` default to `:writer` if the reporting db connection is not defined?

Reply:
I would think we'd want to default to `:reader`, right?